### ISSUE TITLE
Add haplogroup info to Additional Tests (Y-STR)

### ISSUE THREAD

**Issue by lfkennedy** [spawaskar-cora/cora-docs#242]
A separate haplogroup can be requested based on Y-STR data, but there is no place to input that information under "Additional Tests" - see image.

![cora2](https://user-images.githubusercontent.com/35273714/43462749-92f6eaca-949c-11e8-9494-edaadaa2aa09.jpg)

**Comment by clegarde** [comment]
I think it might be helpful to treat "DNA Profile" sort of like the methods in CoRA. You have the option to add it, like you would Age, Sex, etc. You'd have two options: Mito or Nuclear. Mito would include the different types of tests (ex: NGS) and then Nuclear would include Y and auto. But you should be able to add multiple tests for each type (mito or nuclear), since you may conduct more than one (ex for mito: first is inconclusive; ex for nuclear: do Y and auto). Then we could put in the sample number that is appropriate for that particular test along with all the other data that goes along with it (ex: results). I briefly chatted about this with @SachinPawaskarUNO. This would maybe facilitate better searching all the different DNA results that may be associated to a particular skeletal element? 
Just an idea for larger discussion about DNA.

**Comment by lfkennedy** [comment]
@clegarde possibly even dividing this profile out further by descent type may be useful (i.e., mito, Y-chromosome, other nuclear).  This could make it easier for the analyst to decide if elements in a mito sequence need to be resampled/submitted for nuclear, and which test(s) to request.  It may also be nice to have an Associations box under the Y console on the DNA profile page, similar to that for pair matches/articulations, to note shared Y profiles between elements.

**Comment by clegarde** [comment]
Yes, agreed that we should be able to tie samples together based on Y profile. Same would go for autosomal results. The "Mito Sequence Subgroup" can also be used to group in this manner (ex: Oklahoma using a number to represent a Y profile). Although it would be more explicit if we actually linked samples together.

**Comment by CTegtmeyer** [comment]
This has all been updated with the revamp opf the DNA pages.
